esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -4979,7 +4979,7 @@ bool CWsWorkunitsEx::onWUGetZAPInfo(IEspContext &context, IEspWUGetZAPInfoReques
             ipaddr.getHostText(EspIP);
             resp.setESPIPAddress(EspIP.str());
         }
-        if ((version >= 1.96) && !queryRemoteLogAccessor())
+        if ((version >= 1.96) && isContainerized() & !queryRemoteLogAccessor())
             resp.setMessage("Warning: may not be able to include log file because Remote Log Access plug-in is not available!");
```

Comment:
I'm not sure why the wording is "may" ?
If containzerized and there is no log accessor implementation available, then it will not be able to collect logging.
Also, shouldn't the some of the code below this be in the else of this test, i.e. it is appear to still try to get log filenames even though it has established there will be no logging available.

let's chat in Teams to clarify what onWUGetZAPInfo is doing in containerized mode.